etstool.py

```diff
@@ -136,7 +143,6 @@
         'pygments',
     },
     'pyqt5': {
-        'pyqt5',
```

Comment:
Observation: the comment about "should match extras_require in setup.py" at the top of this dict is increasingly inaccurate.